reports/imports/import_2015.py

```diff
@@ -0,0 +1,39 @@
+from .base import BaseImport
+
+from reports.utils.canon import canon
+
+class Import2015(BaseImport):
+    """
+        Holds the methods that handles importing 2015 data files
+    """
+    def get_work_sheet(self, wb, old_sheet, new_sheet, year):
```

Comment:
```suggestion
    def get_work_sheet(self, wb, old_sheet, new_sheet):
```

Import classes already know which year they're dealing with, why do you need to pass it again?